### DISCUSSION TITLE
Localstack support as a quarkus test resource for @QuarkusIntegrationTest

### DISCUSSION THREAD

**Discussion by kirankolli-addepar** [quarkusio/quarkus#33733] [Q&A]
Is there any ready-made support for use of Localstack as a @QuarkusTestResource in a @QuarkusIntegrationTest ? 
The idea being that when running an integration test a LocalStack container is spun up and the appropriate properties injected such that S3 client points to the LocalStack instance instead of the real AWS infra.

[a comment by an automated account is not shown]

**Comment by geoand** [comment]
I don't know of anything personally

**Comment by maxandersen** [comment]
Yes. Localstack is available as devservice when using Amazon extensions. https://quarkiverse.github.io/quarkiverse-docs/quarkus-amazon-services/dev/dev-services.html
